src/api/spec/controllers/webui/packages/meta_controller_spec.rb

```diff
@@ -110,5 +110,41 @@
       it { expect(flash[:error]).to eq('Error while saving the Meta file: fake message.') }
       it { expect(response).to have_http_status(:bad_request) }
     end
+
+    context 'when adding a devel section' do
```

Comment:
Don't test this through a controller spec, it's model code...

Reply:
You're right